### DISCUSSION TITLE
Where can i find the list of all supported methods of a function?

### DISCUSSION THREAD

**Discussion by LucaCVG** [filamentphp/filament#10059] [Help]
Hi, I'm approaching the world of programming and the question is perhaps trivial but I can't find what I'm looking for in the documentation (Filament Docs v3.0). Is there a list of all available methods for a given function?
For example: I would like to know the list of all the methods that can be used for the "TextInput" function of "FormBuilder".

**Comment by joshembling** [comment]
Good question. I would also like to know if this is documented anywhere, without clicking through classes.
